src/mgmt/pool.pb-c.c

```diff
@@ -3133,12 +3133,12 @@ static const ProtobufCFieldDescriptor mgmt__pool_query_resp__field_descriptors[1
     0,NULL,NULL    /* reserved1,reserved2, etc */
   },
   {
-    "total_nodes",
+    "total_engines",
```

Comment:
the proto file src/proto/mgmt/pool.proto needs to be updated with this change and the protobuf shims rebuilt, let me know if you need to do this and push commit to the PR

Reply:
@tanabarr  it would be great if you could do those protobuf updates and push them here - i don't understand how this works, e.g. which files need to be changed and which files don't

Reply:
done

Reply:
also fixed build failures